mewarpx/tests/test_field_diagnostic.py

```diff
@@ -132,6 +132,9 @@ def test_field_diag(plot_on_diag_steps):
     if post_processing:
         print("Verifying that all plots were created...")
         for i in range(0, STEPS - 1, DIAG_STEPS):
+            # at step 0 all data arrays are 0 and are therefore skipped
+            if i == 0:
+                continue
```

Comment:
```suggestion
```suggestion
        # Start at 1st diagnostic (at step 0 all data arrays are 0 and are
        # therefore skipped)
        for i in range(1, STEPS - 1, DIAG_STEPS):
```
```